transforms/transforms.go

```diff
@@ -262,3 +262,49 @@ func GroupBy(identifier RowIdentifier) optimus.TransformFunc {
 		return nil
 	}
 }
+
+// RowFilter is meant to return `true` if a section is meant to be filtered out.
+type RowFilter func(optimus.Row) bool
+
+// passthroughTable is a super simple optimus table that just passes rows through
+type passthroughTable struct {
+	c chan optimus.Row
+}
+
+func (p passthroughTable) Rows() <-chan optimus.Row {
+	return p.c
+}
+func (p passthroughTable) Err() error {
+	return nil
+}
+func (p passthroughTable) Stop() {}
+
+// BypassTransforms effectively wraps a slice of transform funcs with a gate to conditionally
+// apply the transforms only if they match the filter.
+func BypassTransforms(doBypass RowFilter, optionalTransforms []optimus.TransformFunc) optimus.TransformFunc {
```

Comment:
That was a preference for the func name matching the first argument for consistency. I think the other way around is a more intuitive API, but that could be personal experience/preference, so not going to block on it.